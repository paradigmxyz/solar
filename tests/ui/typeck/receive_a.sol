contract C {
    receive() external payable {}
    receive() external payable {}
    //~^ ERROR: receive function already declared
}
