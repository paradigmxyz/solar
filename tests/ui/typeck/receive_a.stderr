error: receive function already declared
  --> ROOT/tests/ui/typeck/receive_a.sol:LL:CC
   |
LL |     receive() external payable {}
   |     ----------------------------- note: previous declaration here
LL |     receive() external payable {}
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |

error: aborting due to 1 previous error

